File: fbmt.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
	int t;
	cin>>t;
	while(t--)
	{
		string s,r,k;
		int n,b=0;
		cin>>n;
		cin>>s;
		for(int i=1;i<n;i++)
		{
			cin>>r;
			if(r!=s){b++;k=r;}
		}
		if(b<n-b)cout<<s<<endl;
		else if(b>n-b)cout<<k<<endl;
		else cout<<"Draw\n";
	}
}

File: fbmt.py
def main():
	t=int(input())
	while t:
		n=int(input())
		a=0
		b=0
		i=0
		s=''
		while i<n: 
			k=input()
			if s==k or i==0: 
				a=a+1
				s=k
			else:
				b=b+1
				w=k
			i=i+1	
		if a>b:print(s)
		elif a<b:print(w)
		else: print('Draw')
		t=t-1 

			
if __name__ == '__main__':
	main()